source/atlas-data-lake-testing/tests/getMore.yml

```diff
@@ -0,0 +1,32 @@
+collection_name: &collection_name "driverdata"
+database_name: &database_name "test"
+
+tests:
+  -
+    description: "A successful find event with getMore"
+    operations:
+      -
+        object: collection
+        name: find
+        arguments:
+          filter: { a: { $gte: 2 }}
+          sort: { a: 1 }
+          batchSize: 3
+          limit: 4
+    expectations:
+      -
+        command_started_event:
+          command:
+            find: *collection_name
+            filter: { a: { $gte : 2 }}
+            sort: { a: 1 }
+            batchSize: 3
+            limit: 4
+          command_name: "find"
+          database_name: *database_name
+      -
+        command_started_event:
+          command:
+            batchSize: 1
+          command_name: "getMore"
+          database_name: "cursors"
```

Comment:
I think there's value in asserting this (as you referenced [RUBY-2211](https://jira.mongodb.org/browse/RUBY-2211)), but let's add a comment above to capture @craiggwilson's comment in https://github.com/mongodb/specifications/pull/821#discussion_r452493691. I suggest adding the following above this line:

```yaml
command_name: "getMore"
# mongohoused always expects getMores on the "cursors" database
database_name: "cursors"
```

Reply:
I have added the comment.